x-pack/platform/packages/shared/kbn-elastic-assistant/impl/assistant/shared_conversation_callout/index.tsx

```diff
@@ -64,6 +64,7 @@ const SharedConversationCalloutComponent: React.FC<Props> = ({
   );
   return localStorageShowConversation && selectedConversation ? (
     <EuiCallOut
+      announceOnMount={false}
```

Comment:
I [see](https://eui.elastic.co/docs/components/display/callout/#EuiCallOut) that `announceOnMount` is `false` by default. Should we add here in this case ?